src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,14 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
 
```

Comment:
Please remove the redundant empty line after the method signature as per the checklist.